src/Quartz/SchedulerBuilder.cs

```diff
@@ -408,6 +406,28 @@ public void UseBinarySerializer()
             {
                 SetProperty("quartz.serializer.type", typeof(T).AssemblyQualifiedNameWithoutVersion());
             }
+
+            /// <summary>
+            /// Configures persistence to use custom ADO provider.
+            /// </summary>
+            /// <typeparam name="TAdoDelegate">Type of the <see cref="StdAdoDelegate"/> inheritor.</typeparam>
+            /// <param name="provider">Valid provider name to configure driver details.</param>
+            /// <param name="configurer">Callback to refine configuration.</param>
+            /// <param name="dataSourceName">Name of the database. Defaults to '<see cref="AdoProviderOptions.DefaultDataSourceName"/>'.</param>
+            public void UseCustomDatabase<TAdoDelegate>(
```

Comment:
I think this should be `UseGenericDatabase` too, it's just an overload with generic support, don't you think?

Reply:
@lahma Yeah.. But introducing overloads is always risky, especially for popular projects like this one, so I chose the safe option.
For example, if you rename UseCustomDatabase, then there will be two methods in the type metadata with the same name and set of parameters, only one will be IsGeneric==true, this can break the code with reflection (if there is one, I don’t know) if this is not taken into account. 
Another option is to make UseCustomDatabase internal.
Maybe I'm overthinking this though..